### PR TITLE
MergeTest

### PR DESCRIPTION
**Pull request by S00185883**

### CODE REVIEW

File: src/components/header/header.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const Header=() => {
+  return (
+    <header className='navbar'>
+      
+        <div className='navbar__title navbar__item'><img src={logo} /></div>
+        <div className='navbar__item'> </div>
+        <div className='navbar__item'> <Popup modal trigger={<button className="info_button"><FaInfoCircle /> Office Re-Entery Requirements</button>}>
+        {close => <Content close={close} />}
+      </Popup></div>
+    </header>
```
Comment (S00189591) [inline src/components/header/header.js:21]: consider cleanup divs?
```suggestion
      
        <div className='navbar__title navbar__item'><img src={logo} /></div>
        <div className='navbar__item'> </div>
        <div className='navbar__item'> 
        <Popup modal trigger={<button className="info_button"><FaInfoCircle /> Office Re-Entery Requirements</button>}>
        {close => <Content close={close} />}
      </Popup>
      </div>
    </header>
```

File: src/components/popup/popup.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    display: block;
+    padding: 2px 5px;
+    line-height: 20px;
+    right: -10px;
+    top: -10px;
+    font-size: 24px;
+    background: #ffffff;
+    border-radius: 18px;
+    border: 1px solid #cfcece;
+  }
```
Comment (S00189591) [inline src/components/popup/popup.css:48]: Again, consider ensuring code is neat and inline

**Review by S00189591 — APPROVED** [review]
Looks good, just consider cleanup some of the code

File: src/components/popup/popup.css (hunk 2 above)
Comment (Clancinio) [reply src/components/popup/popup.css:48]: We should all make sure we are using the prettier extension in VS code so our formatting will be in sync.

File: src/components/popup/entryPopUp.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import '../popup/popup.css'
+import {RiNumber2} from 'react-icons/ri';
+
+
+export default ({ close }) => (
```
Comment (Clancinio) [inline src/components/popup/entryPopUp.js:6]: function EntryPopUp({ close }) {
  return (
    <div className="modal">
       ............code
    </div>
  );
}

**Review by Clancinio — CHANGES_REQUESTED** [review]
Components should be PascalCase :)

File: src/components/popup/entryPopUp.js (hunk 3 above)
Comment (S00185883) [reply src/components/popup/entryPopUp.js:6]: it took this long enough to work i cant risk changing it when i know it works like this

Comment (Clancinio) [reply src/components/popup/entryPopUp.js:6]: Shouldn't effect functionality though. It's just for consistency/convention.

**Review by S00189802 — APPROVED** [review]

**Pull request merged**